chore: Examples implementation and documentation (#5)
terraform-mongodbatlas-modules/terraform-mongodbatlas-atlas-basic: examples/analytics_specs/README.md

# atlas-basic - cluster with analytics specs

> This example shows how the atlas-basic module can be used to create a cluster defining analytic nodes.

Reviewer: shows how you can use the atlas-basic module to create a cluster with analytics nodes

Here and elsewhere: analytics, not analytic

> This example pretends to show how the developer can use the atlas-basic module to create a cluster defining its analytics_specs.

Reviewer 2: ```suggestion
This shows how the atlas-basic module can be used to create a cluster defining analytic nodes.
```
Would go for a more direct description, same phrasing can be applied for other examples if you see fit.

## Usage

> First, you need to set the following variables: 

Reviewer: 1. Set the following variables:

- `org_id`: ID of Atlas organization
- `public_key`: Atlas public key
- `private_key`: Atlas  private key

> Then, to run this example you need to execute:

Reviewer: 2. Run the command as in the following example:
